contributed_definitions/nyaml/NXpeak.yaml

```diff
@@ -1,42 +1,139 @@
 category: base
 doc: |
-  Base class for spectral peaks, their functional form, and support (values).
+  Base class for describing a spectral peak, its functional form, and support values
```

Comment:
We should if not "must" not call this spectral peaks as one can argue that this restricts the base class to be used exclusively for counts = f(energy). However, your approach is already powerful enough to be used also in e.g. mass spectrometry, chromatography or also atom probe tomography methods where clearly we do not measure counts over energy but counts over some other quantity. Still you do peaking fitting and hence your work should be applicable.

Reply:
fixed